member-service/src/main/java/com/hedvig/memberservice/services/customerio/CustomerIOTrustpilotEventListener.kt

```diff
@@ -0,0 +1,97 @@
+package com.hedvig.memberservice.services.customerio
+
+import com.hedvig.integration.notificationService.NotificationService
+import com.hedvig.memberservice.events.EmailUpdatedEvent
+import com.hedvig.memberservice.events.NameUpdatedEvent
+import com.hedvig.memberservice.query.MemberEntity
+import com.hedvig.memberservice.query.MemberRepository
+import com.hedvig.memberservice.services.trustpilot.TrustpilotReviewService
+import com.hedvig.resolver.LocaleResolver
+import com.neovisionaries.i18n.CountryCode
+import org.axonframework.config.ProcessingGroup
+import org.axonframework.eventhandling.EventHandler
+import org.springframework.context.annotation.Profile
+import org.springframework.stereotype.Component
+import java.util.Locale
+import java.util.Objects
+
+@Component
+@Profile("customer.io")
+@ProcessingGroup("CustomerIOTrustpilotBackfill")
```

Comment:
Seems to me like you won't kill off this ProcessingGroup? May as well remove the word "Backfill"

The word "Backfill" to me sounds like it will be removed after being ran. 
```suggestion
@ProcessingGroup("CustomerIOTrustpilot")
```